examples/microprofile/telemetry/greeting/src/main/java/io/helidon/examples/microprofile/telemetry/GreetResource.java

```diff
@@ -48,13 +47,16 @@ public class GreetResource {
 
     private Tracer tracer;
 
+    private io.helidon.tracing.Tracer helidonTracerInjected;
```

Comment:
I do not think this example should be changed. 
Examples are most likely used by new users as a starting point, and this introduces confusion.
We should have an integration test that validates all of this works.
We should have documentation explaining how (and why) to do this.
We may have a new example showing how to combine these APIs.
But I think this example should not be chnaged

Reply:
Fixed